sympy/solvers/solveset.py

```diff
@@ -326,6 +326,52 @@ def _invert_complex(f, g_ys, symbol):
     return (f, g_ys)
 
 
```

Comment:
I don't know how but the `_invert_abs` function was somehow missing. Maybe overlapped by `invert_modular` function. So I just copied the function from my master branch and pasted it here. No changes have been done to the function `_invert_abs`.